appium_lib.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage      = 'https://github.com/appium/ruby_lib' # published as appium_lib
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'selenium-webdriver', '>= 3.0.4', '<= 3.4'
+  s.add_runtime_dependency 'selenium-webdriver', '~> 3.4.3'
```

Comment:
Is there a reason that this needs to be 3.4.3? From what I can tell, all of the proposed fixes in this PR are applicable from 3.4.1 and onwards.